docs/external-references.md

```diff
@@ -102,7 +102,9 @@ It is also a machine-readable record that can be used for analyzing the cross-re
 MyST can integrate directly with other Sphinx documentation, which is used in many Python projects including the [standard library](https://docs.python.org/).
 This re-uses the reference specification defined by [the intersphinx plugin for Sphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html).
 
-Similar to [MyST cross references](#myst-xref), use the `references` object to list Sphinx projects. For example, in the demonstration below we will load the Python 3.7 documentation and Jupyter Book docs, both of which use sphinx and expose cross references through an `objects.inv` file.
+% TODO: Update this when we release Jupyter Book V2 because /stable will no longer be a Sphinx site.
```

Comment:
Note also that the markup is the same - we just specify in the `references` section and it starts looking for myst and falls back to intersphinx. The references will likely have to be updated though as the docs will change.